### ISSUE TITLE
Feature: New UC Folder Process: Project Connect Review (On PROD 🎉🎉)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#8035]
_Requested via email by Isiah L._

**Project Connect Office is joining AULCC as a Committee member and we need to add a review discipline to the UC folder for them.  It shall be named Project Connect Review.  For now @Powell, John and @Robinson, Elizabeth (Beth) are the staff performing this review.  They’d like to have distribution to a group task list.**

**Stakeholder:** Isaiah L., Jaclyn L., Darren U.

**Acceptance Criteria** 
- There need to be a new info field created that displays in all UC folders 
-- Info Description: Project Connect Review?
-- Info type: Boolean
-- Info Group: Distribution
-- Functionality: When Info is answered 'Yes', system needs to insert Project Connect Review Process in UC folders ('Project Connect Review' is a new process and needs to be created) 
-- System should also insert a new document in the Documents tab of the folder called 'Project Connect Review Comments'. The Template for this merge document will be the same as template for other 'Review Comments' documents used in the UC folder. 
- The Process status, attempt results and attempt functionality for the Project Connect Review Process needs to be the same as other review processes (for example - Mobility Bond Process) 
- The Project Connect Review Process needs to be assigned to user 'Project Connect Office' (This is a new user account that needs to be created). 
======================================================================
**Added by Subha ( inputs picked from comment section):**
- USER account  'Project Connect Office' should resemble the user accounts like AWU-Pipeline Engineering or Traffic Control Review, security group can be as in Traffic Control Review and Email no-reply@cityofaustin.org is fine.
- Add another Info '**Project Connect Reviewer**', type Choose, '**Project Connect Office**' should be default value and only value for now in the list.
- - Project Connect Review Process should go into the Process tab AFTER **Initial Distribution**(Similar to the other processes under Distribution in the Info Tab) and insert the Project Connect Review Comments merged document under document tab.
- The TO START and TO END dates also need to come from the PLAN DISTRUBTION DATE(To Start) and AULCC MEETING DATE(To End) In the INFO tab.
- If we do **Adjust Review Due Date process**(61060),  TO END date of Project Connect Review process will be updated with the the process info 'Review Due Date'(51515) of Adjust Review Due Date process(61060).
- Project Connect Review(52300) will be closed and include 'No Response....' comment, when Permit Case Manager Review is Rejected.
- Remove 'Email Notice' attempt for Project Connect Review process. 
-  Project Connect Review (76770) default value should be NO.

**Comment by TracyLinder** [comment]
**Email to Customer:**

Hi Isaih,

Thank you for submitting this need; I've created DTS ticket #8035 to track this request. I'm going to assign this to Vish (copied) to perform scoping/discovery, and get this request ready for development. 

Please be advised that due to the upcoming AMANDA code freeze, we will not be able to push any updates to PROD until the freeze is over. What that means for this request: We'll be able to start development and do UAT, but we won't be able to push this to PROD until after the code freeze, which will be no sooner than late May 2022. 

Please let me know how we can help you work around this block and meet your needs while we wait for the code freeze to end. 

Thank You,
Tracy

**Comment by TracyLinder** [comment]
**Discovery Notes for Vish:**

- UC distribution is based off of the review distribution infos on the info tab. So this will need a new info or two. 
- We can copy the functionality of existing reviews. 
- Don't forget we need to add a review doc to be inserted with the process; same as the other formal reviews on the UC folder. 




FYI @vschandramouli

**Comment by TracyLinder** [comment]
@vschandramouli please begin scoping

**Comment by vschandramouli** [comment]
Notes from scoping meeting 1/25/2022
New Info to be added to folder infos - 
Description - Project Connect Review
Type- Boolean 
Function - When the info is answered Yes, system will insert Project Connect Review Process in UC folder. 
Same functionality as all the other review processes. For example, 'Mobility Bond'.  Attempt functionality will be same. No Process infos, checklists, etc. 
Create new user in AMANDA 6 - Project Connect Office - Assign process to this User. 
System also needs to insert a comments document  like it does for other reviews - Project Connect Review Comments, keep template as previous review documents.

**Comment by vschandramouli** [comment]
Updated AC. Moving to Backlog. 

@Sreelatha-BlazeKannan  @SubhaShroff  @Lpriya-LIT  Can you please review and let me know if you have any questions. I think there may be some items here that are needing clarifications. Please let me know and I will update it as needed. 

CC @Nadin-Nader @TracyLinder

**Comment by TracyLinder** [comment]
@vschandramouli can you please add the stakeholder to the issue description?

**Comment by SubhaShroff** [comment]
@vschandramouli  Here are few questions:
 'Project Connect Office' - Under what Security Group this user should be added? (not necessarily!)
 Any email-id for this user like no-reply@cityofaustin.org?
Any Review fees are involved with the new Process?

**Comment by Nadin-Nader** [comment]
I asked Jaclyn about this and i am waiting on a response

**Comment by Nadin-Nader** [comment]
@SubhaShroff this is the Teams Chat with Jaclyn:

**[2:37 PM] Lozano, Jaclyn**
What security group is the Mobility Bond under? What security group is a process like AT&T(SWBT) 51142 under?
**[2:38 PM] Lozano, Jaclyn**
An email reply? Would that process be sending out an email? No additional fees for that process

**[Yesterday 2:50 PM] Shroff, Subhashini - Vendor**
Mobility Bond is under Review PROCESS group. My question was about PEOPLE 'Project Connect Office' that supposed to be created. It's not mandatory to add any security group or email id.

**[2:44 PM] Lozano, Jaclynz**
Oh does it mean the document when you hit SEND?
**[2:45 PM] Lozano, Jaclyn**
I thought the point of SEND was so that the document would populate on the Master Comment Report when we generate it 
**[2:46 PM] Lozano, Jaclyn**
FYI, the comments from the Project Connect Office Document need to populate on the Master Comment Report. 
**[3:30 PM] Nader, Nadin"**
FYI, the comments from the Project Connect Office Document need to populate on the Master Comment Report." this will probably need to be on a new card.
**[2:50 PM] Shroff, Subhashini -** 
VendorMobility Bond is under Review PROCESS group. My question was about PEOPLE 'Project Connect Office' that supposed to be created. It's not mandatory to add any security group or email id.

**[Yesterday 3:44 PM] Shroff, Subhashini - Vendor**
sorry, I didn't get any of this. For now I leave those questions, as those are not very important to move ahead.

**[Yesterday 7:01 PM] Lozano, Jaclyn**
I would need to double check with Greg. My guess is it would have whatever other PEOPLE accounts like AWU-Pipeline Engineering or Traffic Control Review have. 
**[12:27 PM] Lozano, Jaclyn** 
For security, it should be the same as the Traffic Control Review. And for email, no reply is fine too

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0210803 is submitted for DEV deployment request.

**Comment by Nadin-Nader** [comment]
@SubhaShroff so i had Jacklyn look at the demo and she came back with the following:

> [11:25 AM] Lozano, Jaclyn
> - Change Boolean name to Project Connect Review?- Can you add a drop down menu similar to others under Distribution?      Project Connect Reviewer?                       Project Connect Office is default & only name listed(for now)- Project Connect Review Process should go into the Process tab AFTER Initial Distribution(Similar to the other processes under Distribution in the Info Tab)- The TO START and TO END dates also need to come from the PLAN DISTRUBTION DATE(To Start) and AULCC MEETING DATE(To End) In the INFO tab. - If we do Adjust Review Due Date, will the TO END date be changed as well?- If we do Permit Completeness Check(Reject) Project Connect Review needs to Close(and include similar comment about NO RESPONSE ect.) 
> 
> [11:34 AM] Lozano, Jaclyn
> In the name of the document, UC - Project Connect Review Comments, can you change it to just Project Connect Review Comments

**Comment by Nadin-Nader** [comment]
@SubhaShroff Jaclyn said that the info code for the "Adjust Review Due Date" is **61060**

> [10:20 AM] Lozano, Jaclyn
> 61060 Adjust Review Due Dates.

**Comment by SubhaShroff** [comment]
> @SubhaShroff Jaclyn said that the info code for the "Adjust Review Due Date" is **61060**
> 
> > [10:20 AM] Lozano, Jaclyn
> > 61060 Adjust Review Due Dates.

@Nadin-Nader  After looking into Amanda with the given information, 61060 is the process code of Adjust Reivew Due Dates process. And 51515(Review Due Date) is the process info of this process. 

With the given inputs, summarized and edited the Acceptance Criteria. 

I have sent the demo of the latest changes in email. Please review and let me know.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  DEV deployment request is submitted for the feedback changes - SCTASK0212868.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  Had a quick call with Jaclyn and this what she suggested:
Remove 'Email Notice' attempt for Project Connect Review process. I will do this change.

**Comment by SubhaShroff** [comment]
@Nadin-Nader DEV deployment request is submitted for the changes after talking to Jaclyn to day - SCTASK0213542.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  DEV deployment is done and I tested it for all the changes. Looks good to me now. Kindly, let me know if I can submit TEST deployment request.

**Comment by SubhaShroff** [comment]
@Nadin-Nader TEST manual deployment of the merge document is submitted - SCTASK0213691.
Another TEST deployment request is submitted for the commits - SCTASK0213696.

**Comment by SubhaShroff** [comment]
@Nadin-Nader I tested Merge Document on TEST. It looks as follows and the highlighted part is meant for user phone. Since, we have not given any phone number for 'Project Connect Office' user account, it will always display empty. That is my observation. Needs feedback on it. **Otherwise, its ready for UAT.**
RSN: 12867135
![image](https://user-images.githubusercontent.com/51680715/165770669-5ed61fa3-71b7-449c-8a95-98f41e71d543.png)

**Comment by SubhaShroff** [comment]
### Status:
This ticket is ready for UAT for Jaclyn and her team.

**Comment by Nadin-Nader** [comment]
@SubhaShroff @TracyLinder spoke to Jaclyn and she approved the move to PROD.

Only comment she made was that the process doesnt fall off of the users tasklist after being closed. I informed her that this will have to be addressed in V7

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0216673 - request is submitted for PROD Manual deployment of Merge Document.
SCTASK0216675 - PROD deployment request for the commits is submitted.

**Comment by Nadin-Nader** [comment]
George is assigned to SCTASK0216675, i informed him that SCTASK0216673 is the Merge doc that needs to go with it to PROD

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  This ticket is in PROD now.

**Comment by Nadin-Nader** [comment]
Closing as we have not heard any issues with this so far.
